lib/client/rule.go

```diff
@@ -39,14 +39,25 @@ func ruleActionBackendToAPI(action string) string {
 
 // ruleAPIToBackend converts an API Rule structure to a Backend Rule structure.
 func ruleAPIToBackend(ar api.Rule) model.Rule {
+	var icmpCode, icmpType, notIcmpCode, notIcmpType *int
```

Comment:
I think the naming conventions say `notIcmpCode` should be `notICMPCode`, and the same for type.